<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>connectionstatsaggregated</class>
 <widget class="QWidget" name="connectionstatsaggregated">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1462</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <widget class="QCheckBox" name="remoteNodesCheckbox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Include remote nodes (connects to DOB)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save to file</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="statusLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <column>
      <property name="text">
       <string>Node</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Connection</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>#SentReq</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>#Overflow</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>#Timeout</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>#HandledReq</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>#Overflow</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>#SentMsg</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>#Overflow</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>#RecvMsg</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>#Overflow</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
